modules/examples/include/hephaestus/examples/types/accumulator.h

```diff
@@ -0,0 +1,24 @@
+//=================================================================================================
```

Comment:
What about Kostya's policy of always having a cpp to include the header, even if its empty?
```
// Contributors header
#include "header"
```

Reply:
We actually do, as header files in `examples` or `apps` folder are included in the `.cpp` file of the app. That is needed for library to make sure that somewhere the file is included.